### ISSUE TITLE
Prune and compress images

### ISSUE THREAD

**Issue by heyellieday** [Techtonica/techtonica.org#53]
Upon pulling the repository for the first time, I noticed it was nearly 200mb in size:

![image](https://cloud.githubusercontent.com/assets/7122989/22615532/113ef754-ea4b-11e6-94bb-6668ae157dd2.png)

I think that it would be beneficial to trim the assets stored in the repo, which on disk are roughly 48mb, but I am unsure if that would reduce the size of the repo when initially pulled, based on my initial research: http://stackoverflow.com/questions/8185276/find-size-of-git-repo

If reducing the size of assets store in the repo can reduce the initial size, then an option such as storing the large assets (mostly images) with a static storage provider like Amazon S3 might be worth looking into.

**Comment by hagata** [comment]
This may be helpful, too
http://stevelorek.com/how-to-shrink-a-git-repository.html

It seems that there are a number of images that are not being used on the website that could be cleaned out. Maybe I'm just not seeing them? For example, the `static/workshops/` directory images.

I'm working on #1 & #4 which will include running all the current images through imageoptim.

**Comment by heyellieday** [comment]
@hagata, good to know. I think the work in #1 and #4 combined with using A CDN (https://en.wikipedia.org/wiki/Content_delivery_network) for the compiled assets in production will help the performance on slower clients, but that's probably best discussed in a separate issue.

**Comment by hagata** [comment]
@heyellieday, Yeah, the work for the site performance won't solve this issue.

**Comment by MichelleGlauser** [comment]
Is this the same as [this issue](https://github.com/MichelleGlauser/techtonica/issues/3)?

**Comment by hagata** [comment]
@MichelleGlauser Yes, partially. #3 refers mostly to binary assets, but the root of the issue is the same as this.

**Comment by MichelleGlauser** [comment]
We still do not have a solution for this.

**Comment by gsong** [comment]
There are really two issues here.

## Repo Is Too Large

I recommend we migrate to [git-lfs](https://git-lfs.github.com) to handle this problem. I can convert this repo to use LFS.

## Optimize Images

There's a number of ways to approach this, but the easiest is to probably just do it locally for now when there are new images. On the Mac, just use [ImageOptim](https://imageoptim.com/mac).

**Comment by daaimah123** [comment]
Resolved with [Document and standardize image maintenance within the repo](https://github.com/Techtonica/techtonica.org/issues/503#top)
